.env

```diff
@@ -0,0 +1,6 @@
+NEXT_PUBLIC_FIREBASE_API_KEY=AIzaSyADpeP38ySa1vr_tJIUWVIdEPoNYiTnoZw
```

Comment:
I wouldn't recommend committing .env files especially to a public repo (people can use our firebase credentials for their own application).

Reply:
Client app needs the firebase keys to access Firebase so it doesnt really matter if the keys are public at the moment. We can always create security rules to restrict access.